tests/rkt_os_arch_test.go

```diff
@@ -128,7 +128,7 @@ func getMissingOrInvalidTests(t *testing.T, ctx *testutils.RktRunCtx) []osArchTe
 	invalidArchTest := osArchTest{
 		image:        invalidArchImage,
 		rktCmd:       fmt.Sprintf("%s --insecure-options=image run %s", ctx.Cmd(), invalidArchImage),
-		expectedLine: `bad arch "armv6l"`,
+		expectedLine: `bad arch "armv5l"`,
```

Comment:
I think you can drop this `osArchTest` instance altogether for now. This case was supposed to test fetching a valid ACI image with an arch `rkt` does not support. But after extending the support to more arches, we won't ever get the "bad arch" error for any valid ACI image with os set to linux, because we simply support all the arches for linux that appc/spec supports. The "valid ACI image" is a key here - an image with os/arch pair set to linux/armv5 is an invalid ACI image, so serialization/deserialization of the image manifest happening during fetching will fail.